neon_homeassistant_skill/__init__.py

```diff
@@ -16,39 +17,112 @@ def chunks(lst, n_len) -> List[list]:
+                    )
+                )
+        return self._connected
+
     def initialize(self):
+        try:
+            assert self.connected is True
+        except AssertionError:
+            self.log.warning("Home Assistant PHAL plugin not installed or not running!")
+        if not self.connected:
+            self.on_ready(Message("ovos.phal.plugin.homeassistant.check_connected", None, {"skill_id": self.skill_id}))
+        if self.connected:
+            self.enable_ha_intents()
+
+    def enable_ha_intents(self):
+        self.register_intents()
         self.bus.on("ovos.phal.plugin.homeassistant.assist.message.response", self._handle_assist_error)
         self.bus.on("ovos.phal.plugin.homeassistant.get.device.response", self.handle_get_device_response)
         self.bus.on("ovos.phal.plugin.homeassistant.device.turn_on.response", self.handle_turn_on_response)
         self.bus.on("ovos.phal.plugin.homeassistant.device.turn_off.response", self.handle_turn_off_response)
         self.bus.on(
-            "ovos.phal.plugin.homeassistant.get.light.brightness.response",
-            self.handle_get_light_brightness_response,
-        )
+                "ovos.phal.plugin.homeassistant.get.light.brightness.response",
+                self.handle_get_light_brightness_response,
+            )
         self.bus.on(
-            "ovos.phal.plugin.homeassistant.set.light.brightness.response",
-            self.handle_set_light_brightness_response,
-        )
+                "ovos.phal.plugin.homeassistant.set.light.brightness.response",
+                self.handle_set_light_brightness_response,
+            )
         self.bus.on(
-            "ovos.phal.plugin.homeassistant.increase.light.brightness.response",
-            self.handle_set_light_brightness_response,
-        )
+                "ovos.phal.plugin.homeassistant.increase.light.brightness.response",
+                self.handle_set_light_brightness_response,
+            )
         self.bus.on(
-            "ovos.phal.plugin.homeassistant.decrease.light.brightness.response",
-            self.handle_set_light_brightness_response,
-        )
+                "ovos.phal.plugin.homeassistant.decrease.light.brightness.response",
+                self.handle_set_light_brightness_response,
+            )
         self.bus.on(
-            "ovos.phal.plugin.homeassistant.get.light.color.response",
-            self.handle_get_light_color_response,
-        )
+                "ovos.phal.plugin.homeassistant.get.light.color.response",
+                self.handle_get_light_color_response,
+            )
         self.bus.on(
-            "ovos.phal.plugin.homeassistant.set.light.color.response",
-            self.handle_set_light_color_response,
-        )
+                "ovos.phal.plugin.homeassistant.set.light.color.response",
+                self.handle_set_light_color_response,
+            )
+        self.bus.on("ovos.phal.plugin.homeassistant.ready", self.on_ready)
```

Comment:
Could these be registered in `__init__` or `initialize` unconditionally to avoid accidental re-registration?